doc/source/user_guide/numpy_images.rst

```diff
@@ -147,26 +152,35 @@ denote standard Cartesian coordinates, where ``x`` is the horizontal coordinate,
 ``y`` - the vertical one, and the origin is at the bottom left
 (Matplotlib axes, for example, use this convention).
 
-In the case of multichannel images, the last dimension is used for color channels
-and is denoted by ``channel`` or ``ch``.
+In the case of multichannel images, any dimension can be used for color
+channels, and is denoted by ``channel`` or ``ch``. Prior to scikit-image 0.19,
+the channels axis was always last, but in the current release the channel
+position can be specified by a ``channel_axis`` argument. Functions that
+require multichannel data default to ``channel_axis = -1``. Otherwise,
+functions default to ``channel_axis = None``, indicating that no axis is
+assumed to correspond to channels.
 
 Finally, for volumetric (3D) images, such as videos, magnetic resonance imaging
 (MRI) scans, confocal microscopy, etc. we refer to the leading dimension
 as ``plane``, abbreviated as ``pln`` or ``p``.
 
 These conventions are summarized below:
 
-.. table:: Dimension name and order conventions in scikit-image
+.. table:: *Dimension name and order conventions in scikit-image*
 
-  =========================   ========================================
+  =========================   =============================
   Image type                  Coordinates
-  =========================   ========================================
+  =========================   =============================
   2D grayscale                (row, col)
   2D multichannel (eg. RGB)   (row, col, ch)
   3D grayscale                (pln, row, col)
   3D multichannel             (pln, row, col, ch)
-  =========================   ========================================
+  =========================   =============================
 
+Note that the position of ``ch`` is controlled by the ``channel_axis``
+argument.
+
+|
```

Comment:
Is this RST syntax for a note?

Reply:
I just forces a little extra whitespace in the document. I didn't see any command to add a table footnote, so I was trying to mess with the spacing so that it was clearer that the note above applies to the table.

Reply:
This docs section describes use of pipe (|). The pipe symbol itself doesn't get rendered
https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html#blocks

https://stackoverflow.com/questions/14207327/how-to-insert-blank-line-using-restructuredtext-sphinx

Reply:
Ok, I didn't know, thanks for the tip!